tracer/test/Datadog.Trace.DuckTyping.Tests/StructTests.cs

```diff
@@ -133,5 +133,29 @@ public struct PublicTarget
         {
             public string Name;
         }
+
+        [Fact]
+        public void NonPublicStructCopyFieldTest()
```

Comment:
Can you also add a static field test for this scenario, if there isn't one already?

Reply:
Or does the fix only apply to instance fields?

Reply:
I've added the test case in https://github.com/DataDog/dd-trace-dotnet/pull/4149/commits/06d563571c139e275ebce37e4e5176feda5498a7.